ctapipe/io/tableloader.py

```diff
@@ -529,6 +541,11 @@ def _read_telescope_events_for_id(self, tel_id, start=None, stop=None):
             )
             table = _join_telescope_events(table, impacts)
 
+        if self.interpolate_pointing and POINTING_GROUP in self.h5file.root:
```

Comment:
This looks like it will silently skip doing the requested interpolation if `POINTING_GROUP` is not in the file being processed, but if the user has requested it they would probably like to be at least warned it can't be done.